arrows/vtk/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# Build / Install VTK arrow
+
+set( plugin_vtk_headers
+  depth_utils.h
+  )
+
+kwiver_install_headers(
+  SUBDIR     arrows/vtk
+  ${plugin_vtk_headers}
+  )
+
+kwiver_install_headers(
+  ${CMAKE_CURRENT_BINARY_DIR}/kwiver_algo_vtk_export.h
+  NOPATH   SUBDIR     arrows/vtk
+  )
+
+set( plugin_vtk_sources
+  depth_utils.cxx
+  )
+
+kwiver_add_library( kwiver_algo_vtk
+  ${plugin_vtk_headers}
+  ${plugin_vtk_sources}
+  )
+
+target_link_libraries( kwiver_algo_vtk
+  PUBLIC               vital
+  PRIVATE              vtkFiltersSources
```

Comment:
I think vtkFiltersSources needs to be PUBLIC since it's references in the header, depth_utils.h, thus exposed in the public API.

Reply:
I wasn't directly using the right libraries either.  It's fixed now.

Reply:
Thanks ... I wondered about that as well since the libraries didn't seem to match up with the headers but since I don't know much about VTK library organization, I took that one on faith :)

Reply:
Yeah, I don't know a good way of figuring out which VTK headers belong to which libraries other than by going into the VTK source code and searching for header file names to figure out where they came from, which is what I had to do here.